barretenberg/cpp/src/barretenberg/stdlib/primitives/bool/bool.cpp

```diff
@@ -438,12 +438,6 @@ template <typename Builder> bool_t<Builder> bool_t<Builder>::implies(const bool_
     return (!(*this) || other); // P => Q is equiv. to !P || Q (not(P) or Q).
 }
 
-/**
```

Comment:
Did this go away because it's incorrect?

Reply:
This comment seemed to have been duplicated (maybe wrong merge/rebase resolution). The very same comment is on top of the next method and in this case the comment makes sense. Here, I did not see the point and I was misled until I realized that it is a duplicated comment.